File: 3.greeting.c
#include<stdio.h>  
#include <string.h>

int main()  
{
  char name[40];  
  printf("Enter Your Name : "); 
  scanf("%s",&name);  
  if(strcmp(name,"Bob") == 0 || strcmp(name,"Alice") == 0){
    printf("Hello, %s", name);  
  } else {
    printf("Hello");
  }
}

File: 3.greeting.cs
using System;

class Program{
    static void Main(){
      Console.WriteLine("Enter your name:");
      string userName = Console.ReadLine();
      if(userName == "Bob" || userName == "Alice"){
        Console.WriteLine("Hello, " + userName); 
      } else {
        Console.WriteLine("Hello");
      }
    }
}